### ISSUE TITLE
frient Smart Plug Mini 2 (SPLZB-141)

### ISSUE THREAD

**Issue by fadamsen** [dresden-elektronik/deconz-rest-plugin#7187]
## Device

- Product name: Smart Plug Mini 2
- Manufacturer: frient A/S
- Model identifier: SPLZB-141
- Device type:
  - Switch

The device shows up as a light, but should be a switch. Sensors for power and consumption are not added.

### Basic
![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/4189385/3d5e9ef9-0c48-4a40-bd21-66ded5da6734)

![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/4189385/d4559b58-7d8f-4e3e-8283-fcf641697fc5)

### Simple Metering
![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/4189385/425455dc-8c57-4cda-a889-4baa6924f74c)

![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/4189385/8d534292-7c23-4c2c-befd-260c576e0b34)

### Electrical Measurement
![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/4189385/e2b9458d-5859-4edd-90e8-9c8a5de812ce)

![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/4189385/f38f1c40-9184-4d1a-9801-299b27f5860a)

![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/4189385/0f7cb34c-a533-4805-9c6f-4a9bde427a3a)

![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/4189385/cbff0db3-5ddc-424c-9bb9-6a0eb98808c2)

![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/4189385/39419206-9921-466a-8b42-cf8a3c9a5e90)

**Comment by SwoopX** [comment]
You could modifiy the current DDF for the old version (file `splzb-131_smart_plug.json` in the frient folder) and change the following lines from

```
  "manufacturername": ["frient A/S", "frient A/S"],
  "modelid": ["SPLZB-131", "SPLZB-134"],
```

to

```
  "manufacturername": ["frient A/S", "frient A/S", "frient A/S"],
  "modelid": ["SPLZB-131", "SPLZB-134", "SPLZB-141"],
```

**Comment by fadamsen** [comment]
@SwoopX, I've tried that, but Deconz doesn't pick up the change. I'm running Deconz through Home Assistant as an addon (latest version of both), and despite being able to open the modified DDF file and issuing the Force Reload command, nothing happens.

**Comment by Smanar** [comment]
Select the node on the GUI, and select "edit DDF", if it's not the good one that appear there is a problem (you will the path and the name on the window title)

On HA https://forum.phoscon.de/t/how-to-add-edit-a-ddf-on-home-assistant-using-text-editor/1839

**Comment by fadamsen** [comment]
I managed to make it work! It might have been an issue with line endings, in any case it was solved when I used a copy of `/usr/share/deCONZ/devices/frient/splzb-131_smart_plug.json` and placed it in `/data/.local/share/dresden-elektronik/deCONZ/devices`. That directory survives restarts and updates of the deCONZ addon, so I'm good for now.

I see @SwoopX already went ahead and created #7189 to merge the change, which is great, so everyone else will be able to use the smart plug without having to mess with DDFs.
